docs/aks/README.md

```diff
@@ -15,22 +15,27 @@ Make sure only the following permissions are assigned to the key pair:
  * Event streams: READ
  * Hosts: READ
 
-### Step 2: Create new Azure credentials
+### Step 2: Configure Azure and Log Analytics
 
-TBD TODO
+Sign in to the Azure portal at [https://portal.azure.com/](https://portal.azure.com/)
 
-```
-WORKSPACE_ID=ASDF
-```
+1. Create a new resource group by selecting `Resource groups`.
+1. Select `New`
+1. Enter a new resource group name
+1. Select `Review + Create`
+1. Select `Create`
 
-```
-PRIMARY_KEY=ASDF
-```
-### Step 3: Set-up Azure Log Analytics
+1. Create a new Log Analytics workspace by selecting `Log Analytics workspaces`. You might have to search for it as a resource if it is not in your home menu.
+1. Select `New`
+1. Select the resource group that was previously created and enter a name under `Instance details`
+1. Select `Review + Create`
+1. Select `Create`
+
+Assign it to your resource group. (Home > Log Analytics workspaces > Create)
```

Comment:
I think these lines need to be dropped. They appear to be duplicates of the instructions in line 28-32, but once they are, I think we should merge!